projects/cdk/schematics/utils/templates/elements.ts

```diff
@@ -1,5 +1,7 @@
 import {ChildNode, Element, parseFragment} from 'parse5';
 
+const ALWAYS_TRUE_HANDLER = (): true => true;
```

Comment:
Yes, it is
https://github.com/Tinkoff/taiga-ui/blob/f30ab52da856a426d99cbe43b8c1a91e72689c19/projects/cdk/constants/always-true-handler.ts#L4

But i cannot use it inside schematics because of these lines((
https://github.com/Tinkoff/taiga-ui/blob/f30ab52da856a426d99cbe43b8c1a91e72689c19/tsconfig.schematics.json#L12